src/dist/replication/meta_server/meta_bulk_load_service.cpp

```diff
@@ -497,6 +561,171 @@ void bulk_load_service::handle_app_unavailable(int32_t app_id, const std::string
     // TODO(heyuchen): TBD
 }
 
+// ThreadPool: THREAD_POOL_META_STATE
+void bulk_load_service::update_partition_metadata_on_remote_stroage(
+    const std::string &app_name, const gpid &pid, const bulk_load_metadata &metadata)
+{
+    zauto_read_lock l(_lock);
+    partition_bulk_load_info pinfo = _partition_bulk_load_info[pid];
+    pinfo.metadata = metadata;
+    blob value = json::json_forwarder<partition_bulk_load_info>::encode(pinfo);
+
+    _meta_svc->get_meta_storage()->set_data(
```

Comment:
Is 'remote_stroage' zookeeper here?

Reply:
Yes, we use `remote_storage` to represent zk.